src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const myList = document.querySelector('.tree');
+
+const listItems = myList.getElementsByTagName('li');
+
+for (let i = 0; i < listItems.length; i++) {
+  listItems[i].hidden = false;
+
+  const newSpan = document.createElement('span');
+
+  listItems[i].insertAdjacentElement('beforebegin', newSpan);
+  newSpan.append(listItems[i]);
+}
+
+const add = (e) => {
+  if (!e.target.children.length || e.target.tagName === 'UL') {
```

Comment:
Check if `e.target.tagName` is 'SPAN' instead of checking if `e.target.children.length` is not zero or `e.target.tagName` is not 'UL'.